src/centrality.rs

```diff
@@ -132,3 +132,48 @@ pub fn digraph_betweenness_centrality(
             .collect(),
     }
 }
+
+/// Compute the closeness centrality of all nodes in a PyGraph.
+///
+/// :param PyGraph graph: The input graph
+/// :param bool normalized: Whether to normalize the betweenness scores by the number of distinct
+///    paths between all pairs of nodes.
+/// :param bool endpoints: Whether to include the endpoints of paths in pathlengths used to
+///    compute the betweenness.
+/// :param int parallel_threshold: The number of nodes to calculate the
+///     the betweenness centrality in parallel at if the number of nodes in
+///     the graph is less than this value it will run in a single thread. The
+///     default value is 50
```

Comment:
Looks like a copy paste issue, we should also add the docstring for `wf_improved`

```suggestion
```